iron-location.html

```diff
@@ -209,14 +209,17 @@
         // nothing to do, the URL didn't change
         return;
       }
+      // Need to use a full URL in case the containing page has a base URI.
+      var fullNewUrl =
+          window.location.protocol + '//' + window.location.host + newUrl;
```

Comment:
I think you might be able to do a slightly more robust approach here with the [`URL`](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL) class. For example:
```
var fullNewUrl = new URL('foo', window.location.origin).href;
```